scripts/serve-local.sh

```diff
@@ -10,5 +10,6 @@ docker run \
     --name janis \
     --tty --interactive \
     --publish 3000:3000 \
-    --volume "$PWD:/app" \
+    --volume "$PWD/src:/app/src" \
+    --volume "$PWD/public:/app/public" \
```

Comment:
Does it make sense to put `package.json` in there as well? Most likely you'd need to rebuild if you add a package, but there might be other changes that wouldn't require a rebuild.

While we're talking about it, I'm also wondering if we need to copy `yarn.lock` in `Dockerfile`?

Reply:
I'll add these.